src/middlewares/validator.js

```diff
@@ -0,0 +1,30 @@
+import { validationResult } from 'express-validator/check';
+import { matchedData } from 'express-validator/filter';
+
+/**
+ * Generic schema validator
+ *
+ * @param {Array} schema
+ * @param {number} status
+ *
+ * @returns {object} returns response message
+ */
+
+export default (schemas, status = 400) => {
+  const schemaTovalidate = schemas || [];
+
+  const validationCheck = async (request, response, next) => {
+    const errors = validationResult(request);
+    request = { ...request, ...matchedData(request) };
+
+    if (!errors.isEmpty()) {
+      const mappedErrors = errors.mapped();
+
+      return response.status(status).json(mappedErrors);
```

Comment:
What format is this error in? Please make sure it follows our convention and drop a snippet of the format here